src/content/advanced/security-policy-enforcement/policy-api.md

```diff
@@ -0,0 +1,69 @@
+Among these are a number of resources designed for enforcing policies within a cluster. These include built-in types like Network Policies and RBAC as well as external CRDs like Kyverno Cluster Policies or Cilium Network Policies, among others.
+
+We use and manage tools which we see as bringing value for customers, and which are governed and maintained in a way which aligns with our guiding principles.
+We encourage our customers to optimize, experiment, and make full use of all of our platform components, but we also recognize that using many different tools comes with overhead and complexity.
+We want to make that overhead optional, and be able to fully manage a feature without requiring customers to change tools, update config, or plan for complicated migrations when one of the underlying implementations changes.
+
+For example, when the Pod Security Policy type was deprecated and removed, many workloads had to be re-evaluated and new exceptions created for them, even though neither the workload nor the risk had changed.
+Some new features were gained, but the effort was primarily invested in order to maintain feature parity and avoid security regressions.
+We expect that Kubernetes and third-party tooling will continue to evolve, and that we can help our customers be faster if they are not directly tied to tool-specific interfaces which they don't actually want to manage.
+
+So, we created the Policy API in order to allow Giant Swarm to more seamlessly and transparently move clusters between policy implementations, and to reduce the overall toil of dealing with common security configuration.
+
+### At a glance
+
+The Policy API:
+
+- is an interface for configuring the various types of (mostly security-related) policies which Giant Swarm manages.
+- provides a way for cluster administrators to declare their intent about which policies to enforce and which resources are exempt from those policies.
+- is intended to manage additional policy types in the future, including networking, vulnerability management, anomaly detection, and others.
+- is _not_ a general purpose policy syntax language. Users cannot define custom policies via the Policy API. It can only be used to configure the policies Giant Swarm actively manages.
+- does _not_ hide the underlying implementations. Users are free to directly use the underlying tools or APIs. The only difference is that _Giant Swarm will not manage, migrate, or adopt any policies or configuration passed directly_.
+- generates native resources for the underlying implementations. These resources continue to function even if the Policy API controllers are removed.
+  
+#### Managed and un-managed policies
+
+The Policy API only orchestrates policies which Giant Swarm actively manages. It does not interfere with customer policies, exceptions, or configuration which are managed externally to the Policy API.
```

Comment:
> The Policy API only orchestrates policies which Giant Swarm actively manages.

As a customer, I wouldn't really get the point. Rather than saying "Giant Swarm manages something", can we describe it as "Giant Swarm provides a set of ready-made policies to choose from"? At least that's how I understand this works. And by now, we're already a few minutes into someone reading the article, so let's put an example here.

Reply:
Well, the next sentence is an example. But I can reword this
```suggestion
Giant Swarm provides a set of ready-made policies for many common cluster management use cases. The Policy API only orchestrates these standard policies, which Giant Swarm actively manages. It does not interfere with customer policies, exceptions, or configuration which are managed externally to the Policy API.
```